api/rest/v1/bulk_find_entities.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: default
+title: Find DCIDs
+parent: REST (v1)
+grand_parent: API
+nav_order: 1
+published: true
+permalink: /api/rest/v1/bulk/find/entities
+---
+
+# /v1/bulk/find/entities
+
+Find the [DCID](/glossary.html#dcid) of an entity.
```

Comment:
Firstly, for a given entity, we might find multiple possible DCIDs, not just "the" DCID.

Secondly, this is a bulk method, so this is not for "an entity", but for a list of entities. This PR is for a single entity: https://github.com/datacommonsorg/mixer/pull/983

Reply:
Reworded.